tlscanary/js/scan_worker.js

```diff
@@ -122,8 +122,12 @@ function collect_request_info(xhr, report_certs) {
         sec_info.QueryInterface(Ci.nsITransportSecurityInfo);
         info.transport_security_info_status = true;
         info.security_state = sec_info.securityState;
-        info.security_description = sec_info.shortSecurityDescription;
         info.raw_error = sec_info.errorMessage;
+        try {
+            info.short_error_message = info.raw_error.split("Error code:")[1].split(">")[1].split("<")[0];
```

Comment:
How sure are we that the format remains stable a) over time and b) across messages? Slight string changes might lead to capturing the wrong data instead of throwing an error.

Reply:
Well, until we have a better API, we need something. Otherwise we'd have to display a very long error string in the data grid, which would look bad.

Also, this logic is identical to what we already have in report.py, line 236, so if it breaks here, it will break there as well. At least it's in a try/catch statement, in the event that the format completely changes and the parsing fails.

Reply:
Alright, all good to land then.